### ISSUE TITLE
Unpredictable behavior with several config files

### ISSUE THREAD

**Issue by RealElias** [bootique/bootique#230]
Application choose random config file if several modules have config.yml files for same path in classpath

**Comment by IRus** [comment]
Thanks for submitting issue. Fix for this issue will be shipped in Bootique 0.26.

**Comment by IRus** [comment]
https://github.com/bootique/bootique/commit/5156bb502dd03411f872e60735d41b6beec60959#commitcomment-29516653 

Travis is green, @andrus what tests exactly breaks this PR?

**Comment by andrus** [comment]
It broke the tests in the app using 0.26-SNAPSHOT.

**Comment by andrus** [comment]
Nope, that's a straightforward test with "autoLoadModules" on. The failing file is coming from the upstream `.jar`. The problem occurs when running `mvn clean verify` from command line. It works in Idea. So I think I know what happens:

* The tested app is an app, so it is packaged as a fat jar. So the YAML file from another jar is packaged in it.
* But the test also has that jar on classpath. Hence it ends up being included twice. 

Don't see a way around with fat jar packaging.

**Comment by IRus** [comment]
Maybe we can compare content of resources, and fail only if content differs.

**Comment by andrus** [comment]
Or maybe we just print a warning via BootLogger and let it continue without an exception.
